src/services/Friend/HandleGroup.js

```diff
@@ -92,11 +115,34 @@ export async function removeUserFromGroup(groupID, userID, app = firebase) {
   batch.delete(userRef.collection("groupJoined").doc(groupID));
   batch.delete(groupRef.collection("members").doc(userID));
 
+  const username = await db
+    .collection("users")
+    .doc(userID)
+    .get()
+    .then((doc) => {
+      console.log(doc.data());
+      return doc.data().username;
+    });
+
+  const groupName = await db
+    .collection("groups")
+    .doc(groupID)
+    .get()
+    .then((doc) => doc.data().name);
+
   await batch
     .commit()
     .then(() => {
       Alert.alert("This user has been removed from the group.");
     })
+    .then(() => {
+      sendNotificationToAllGroupMembers(
+        groupID,
+        userID,
+        "Chattoku's Group Member Removed",
+        `${username} has been removed from ${groupName} by the group admin.`
+      );
```

Comment:
I think this notification is only necessary for the removed user, not every group member